packages/resilience/src/resilience.spec.ts

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import nock from 'nock';
+import { executeWithMiddleware } from './middleware';
+import { retry } from './retry';
+import { timeout } from './timeout';
+
+describe('combined resilience features', () => {
+  const HTTP_STATUS = {
+    OK: 200,
+    UNAUTHORIZED: 401,
+    FORBIDDEN: 403,
+    SERVICE_UNAVAILABLE: 504
+  };
+
+  it('Needs to retry with delay below timeout', async () => {
```

Comment:
[nth] start with lower case in the sentence so the `it` - applies also for the other tests.